README.md

```diff
@@ -7,43 +7,40 @@
 [![dev chat](https://discordapp.com/api/guilds/299006062323826688/widget.png?style=shield)](https://discord.gg/ga2xZXk)
 
 
-The source code of `karaoke!` ruleset, running on [osu!lazer](https://github.com/ppy/osu).
+The source code of the `karaoke!` ruleset, for [osu!lazer](https://github.com/ppy/osu).
 
 ## Status
 
-Some feature is still heavy developing, but exist feature should work fine.
+Some features are still under heavy development, but existing features should work fine.
 
-If got any problem, you can just ask me in gmail(andy840119@gmail.com) or [discord channel](https://discord.gg/ga2xZXk)(will reply faster) instead of costing too much time in solving problem.
+If you run into any problems, you can shoot me an email (andy840119@gmail.com) or send me a message on [discord](https://discord.gg/ga2xZXk). I will typically reply faster on discord.
 
-And feel free to report bugs if found.
+And feel free to report any bugs if found.
 
 ## How to run this project
 
-See [this tutorial](https://karaoke-dev.github.io/how-to-install/README.html) to get ruleset from exist build.
+See [this tutorial](https://karaoke-dev.github.io/how-to-install/README.html) to get the ruleset from the existing build.
 
-Or you can compile it by yourself, `release build` then copy `osu.Game.Rulesets.Karaoke.Packed.dll` into [ruleset folder](https://github.com/LumpBloom7/sentakki/wiki/Ruleset-installation-guide)
+Or you can compile it yourself: `release build` then copy `osu.Game.Rulesets.Karaoke.Packed.dll` into your [ruleset folder](https://github.com/LumpBloom7/sentakki/wiki/Ruleset-installation-guide)
 
-If wants to try on other platform like android or ios, you might use [launcher](https://github.com/karaoke-dev/launcher).    
-Notice : The launcher is still under development, so it might not work.
+If you want to try the ruleset on another platform like android or ios, you would use [launcher](https://github.com/karaoke-dev/launcher).    
+Notice : The launcher is still under development, so it might not work as intended.
 
 ## License
 
-This repo is under [GPL V3](LICENSE) license.
-
-## Rules
-
-1. If you plan on using this repo for commercial purposes, please contact me(andy840119@gmail.com) to get permission first.
-2. Use this repo to create/use `PIRATED` karaoke beatmap is absolutely forbidden.
+This repo is covered under the [GPL V3](LICENSE) license.
+If you plan on using this repo for commercial purposes, please contact me at (andy840119@gmail.com) to get permission first.
+Using this repo to create a `PIRATED` karaoke ruleset is absolutely forbidden.
 
 ## Thanks to
 
-- [osu!](https://github.com/ppy/osu) and it's [framework](https://github.com/ppy/osu-framework), karaoke! is modified from them.
+- [osu!](https://github.com/ppy/osu) as it's [framework](https://github.com/ppy/osu-framework) for karaoke!
```

Comment:
osu! is based on osu-framework, i'm not sure can use `as`?

Reply:
but osu-framework is created because osu!lazer's need 🤔

Reply:
I think that it’s ok to say this because karaoke is made on lazer which is made on osu framework

Reply:
Hmm...
I'm poor on grammar, but if this means that it's ok.

Reply:
After realizing that people might make karaoke run on another project based off the osu framework, you were right to put “and” here so I changed it back